src/bin/bpf-linker.rs

```diff
@@ -118,10 +118,6 @@ struct CommandLine {
     #[clap(long)]
     allow_bpf_trap: bool,
 
-    /// Add a directory to the library search path
```

Comment:
why this change? bpf-linker needs to stay compatible with rustc, we can't change
the CLI

Reply:
ok got it, it is because of rustc, I will revert it

I removed it because it was unused